static/app/views/projectDetail/projectQuickLinks.tsx

```diff
@@ -50,6 +50,14 @@ function ProjectQuickLinks({organization, project, location}: Props) {
         query: {project: project?.id},
       },
     },
+    {
+      title: t('Key Transactions'),
```

Comment:
This isn't actually called "Key Transactions". 

```suggestion
      title: t('View Transactions'),
```